src/plugins/intel_cpu/src/nodes/subgraph.cpp

```diff
@@ -499,7 +501,22 @@ void Snippet::SnippetJitExecutor::schedule_6d(const std::vector<MemoryPtr>& inMe
             int64_t indexes[] = {d0, d1, d2, d3, d4};
             jit_snippets_call_args call_args;
             update_ptrs(call_args, inMemPtrs, outMemPtrs);
+#ifdef __linux__
+            if (g_enable_snippets_err_detector) {
+                __sighandler_t signal_handler = [](int signal) {
+                    ov::intel_cpu::g_snippets_err_handler->local()->print_debug_info();
+                    OPENVINO_THROW("Segfault was caught by the signal handler");
```

Comment:
> Could you please add an information that the segfault happened during Snippets node execution? 

done, thanks!

Reply:
> We can even print node name if I am not mistaken

Let's discuss on the way to print node name in [comment](https://github.com/openvinotoolkit/openvino/pull/19935#discussion_r1384465179)

Reply:
Mu point was about printing CPU node name here: can we capture `this` in `segfault_handler` and add `this->getName()` to the exception message? Smth like
`OPENVINO_THROW("Segfault was caught by the signal handler during execution of Snippets node with name ", this->getName());`

Reply:
unfortunately the segfault_handler signature is fixed and does not accept this pointer.

Reply:
Got it, thanks for the explanation.
Another idea: we can theoretically catch the exception from segfault detector in `Snippets::execute` method and rethrow it with extended message. However, I could miss something

Reply:
c++ exception(try catch) and single handler is not compatible, see https://stackoverflow.com/questions/1717991/throwing-an-exception-from-within-a-signal-handler/1718124#1718124.
Set up a global variable is a approach that segfault_handler can access to provide additional info.
done in [075aae9](https://github.com/openvinotoolkit/openvino/pull/19935/commits/075aae97a598fa3e4c84329ba74ff6ddfcc56fa4)

Reply:
Thanks a lot for the justification. I would prefer not print subgraph node name at all to avoid creating another global variable for this purpose.
I suppose, in case of debugging it will not be difficult to manually put prints in the execute to identify a falling Subgraph.

Reply:
agree, the global variable(not thread local) has risk, for example when extend to support subgraph execution in parallel. removed.